db/migrate/20210428151759_create_followers.rb

```diff
@@ -0,0 +1,10 @@
+class CreateFollowers < ActiveRecord::Migration[6.1]
+  def change
+    create_table :followers do |t|
+      t.references :user, null: false, foreign_key: true
+      t.bigint :follower
```

Comment:
Why is `follower` not a foreign key?

Reply:
I only added for models now i add the migration for the database.
```
class AddForeignToFollowers < ActiveRecord::Migration[6.1]
  def change
    add_foreign_key :followers, :users, column: :follower_id
  end
end
```